packages/bpk-component-calendar/src/bpk-calendar-date.scss

```diff
@@ -54,15 +57,18 @@
   }
 
   &--selected {
-    background-color: $bpk-state-selected-background-color;
-    color: $bpk-calendar-day-selected-color;
     font-weight: bold;
     cursor: default;
+
+    @include bpk-themeable-property(background-color, --bpk-calendar-date-selected-background-color, $bpk-state-selected-background-color);
+    @include bpk-themeable-property(color, --bpk-calendar-date-text-selected-color, $bpk-calendar-day-selected-color);
   }
 
   &--focused:not(:disabled):not(.bpk-calendar-date--selected) {
-    color: $bpk-state-selected-background-color;
     box-shadow: 0 0 0 2px $bpk-state-selected-background-color inset;
+    box-shadow: 0 0 0 2px var(--bpk-calendar-date-focused-border-color, $bpk-state-selected-background-color) inset;
```

Comment:
this looks like a copypasta error, is the box shadow themable?

Reply:
It's intentional. `box-shadow` is too advanced to use `bpk-themeable-property`. The above line is a fallback for browsers that don't support CSS variables.